kafka-rest/src/main/java/io/confluent/kafkarest/KafkaRestConfig.java

```diff
@@ -709,6 +729,24 @@ public Properties getOriginalProperties() {
     return properties;
   }
 
+  public final Map<String, Object> getSchemaRegistryConfigs() {
+    HashMap<String, Object> possibleConfigs = new HashMap<>();
+    possibleConfigs.putAll(originals());
+    possibleConfigs.putAll(originalsWithPrefix("schema.registry.", /* strip= */ true));
+    possibleConfigs.putAll(originalsWithPrefix("client.", /* strip= */ true));
+    possibleConfigs.putAll(originalsWithPrefix("producer.", /* strip= */ true));
+    possibleConfigs.putAll(originalsWithPrefix("consumer.", /* strip= */ true));
+
+    // Disable auto-registration of schemas. Schema registration is handled by SchemaManager.
+    possibleConfigs.put(AUTO_REGISTER_SCHEMAS, false);
+    // Disable latest-version fetching of schemas.
+    possibleConfigs.put(USE_LATEST_VERSION, false);
```

Comment:
again, seems a bit sloppy - allowing the user to specify SR config, then silently ignoring it.

also, why make it so the user can't turn off auto schema registration? surely that's a feature people would want? is it a breaking v2 change?

also, why arbitrarily disable `use.latest.version` capability? is this a breaking change from current v2 capability?

Reply:
Read https://github.com/confluentinc/kafka-rest/pull/787#discussion_r558461865 first :)

This configurations don't make much sense in REST Proxy, because the behaviour controlled by them is entirely driven by the API. The REST Proxy behaviour has as been:
* if you specify `schema_id`, use the respective schema
* otherwise, use `schema` and we will auto-register it IF it doesn't exist (in the same subject) yet.

There's has never been a way of using latest version for a subject (in V2, although there will be in V3).

There have been many tickets of people setting these configurations, and then being surprised things don't work as expected. I think it is better to explicitly disable them and update the documentation.

Reply:
sounds good (if not a regression)